components/LinkCardComponent.tsx

```diff
@@ -0,0 +1,79 @@
+import React, { useState } from "react";
+import styles from "../styles/FolderPage.module.css";
+import { formatDate, setBeforeDate } from "../utils/commonUtil";
+import { styled } from "styled-components";
+import CardDropdownComponent from "./CardDropdownComponent";
+import Link from "next/link";
+
+type props = {
+  imgSrc?: string;
+  createdAt: string;
+  desc?: string;
+  url: string;
+  type?: "default" | "static";
+};
+
+const LinkCardComponent = ({
+  imgSrc,
+  createdAt,
+  desc,
+  url,
+  type = "default",
+}: props) => {
+  const [show, setShow] = useState(false);
+
+  const curDate = new Date(createdAt);
+  const noImage = "/images/card_no_img.png";
+
+  return (
+    <Link href={url} target="_blank" className={styles.card_link}>
+      <div className={styles.folder_card}>
+        <img src={imgSrc ?? noImage} className={styles.card_img} />
+        <div className={styles.card_info}>
+          <p className={styles.card_date_before}>{setBeforeDate(curDate)}</p>
+          <p className={styles.card_desc}>{desc}</p>
+          <p className={styles.card_date}>{formatDate(curDate)}</p>
+        </div>
+      </div>
+      <div className="link_absolute_div">
+        {type === "default" ? (
+          <LinkStar>
+            <img src="/images/star.svg" />
+          </LinkStar>
+        ) : (
+          <></>
+        )}
+        {type === "default" ? (
+          <KebabButton
+            onClick={(e) => {
+              e.preventDefault();
+              setShow(!show);
```

Comment:
이렇게 표현해주는게 좋을 것 같네요!
```suggestion
              setShow(prev => !prev);
```